assets/src/scripts/selectors/flood-data-selector.spec.js

```diff
@@ -78,6 +79,87 @@ describe('flood-data-selector', () => {
         });
     });
 
+       describe('hasWaterwatchData', () => {
+        it('Return false if no waterwatch flood levels are available', () =>{
+            expect(hasWaterwatchData({
+                floodState: {
+                    actionStage: null,
+                    floodStage: null,
+                    moderateFloodStage: null,
+                    majorFloodStage: null
+                }
+            })).toBeFalsy();
+        });
+
+        it('return true if waterwatch flood levels are available', () => {
+            expect(hasWaterwatchData({
+                floodState: {
+                    actionStage: 1,
+                    floodStage: null,
+                    moderateFloodStage: null,
+                    majorFloodStage: null
+                }
+            })).toBeTruthy();
+        });
+    });
+
+       describe('getWaterwatchData', () => {
+        it('Return true if waterwatch flood levels are returned', () =>{
+            expect(getWaterwatchFloodLevels({
+                floodState: {
+                    actionStage: 1,
+                    floodStage: 2,
+                    moderateFloodStage: 3,
+                    majorFloodStage: 4
+                }
+            })).toEqual([1,2,3,4]);
+        });
+    });
+
+       describe('waterwatchVisible', () => {
+        it('Return false if waterwatch flood levels should not be visible', () =>{
```

Comment:
I'd like a test for when the parameter code is 00065 but there are no flood levels.

Reply:
Added test for correct param code but no flood levels.